oaknational/Oak-Web-Application
# build: add config push/pull scripts, add webinar url to config

## Description

- `webinarSignUpUrl` to write_env_file and configs
- creates `push_config` and `pull_config` scripts

## Issue(s)

Progresses https://github.com/oaknational/Oak-Web-Application/issues/978
Progresses https://github.com/oaknational/Oak-Web-Application/issues/355

## How to test

2. Run: `npm run config:pull`
3. Should pull config from cloud bucket to `oak-config` folder
4. Run: `npm run config:push:{dev|prod}` 
5. Should display diff of your local file and the live file, asking you to confirm changes if there are any
6. If confirmed, should push your local config live

## Checklist
